gnomic/genotype.py

```diff
@@ -261,7 +261,7 @@ def changes(self):
 class Genotype(object):
     def __init__(self, changes, parent=None):
         if parent:
-            state = GenotypeState(parent.state.changes)
```

Comment:
`GenotypeState` is expecting a list of tuple like `[(before, after), (before, after)]` to initialize `self._changes`

`self.state.changes` is a list of change like `[Change(before, after), ...]`